examples/workos-integration-demo/.env.local.example

```
@@ -0,0 +1,10 @@
+# Set NEXT_PUBLIC_HEXCLAVE_PORT_PREFIX to use the repository's alternate port prefix.
+NEXT_PUBLIC_HEXCLAVE_API_URL=http://localhost:8102
+NEXT_PUBLIC_HEXCLAVE_PROJECT_ID=internal
+NEXT_PUBLIC_HEXCLAVE_PUBLISHABLE_CLIENT_KEY=this-publishable-client-key-is-for-local-development-only
+HEXCLAVE_SECRET_SERVER_KEY=this-secret-server-key-is-for-local-development-only
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HEXCLAVE_SECRET_SERVER_KEY key should go before the NEXT_PUBLIC_HEXCLAVE_API_URL key

(UnorderedKey)